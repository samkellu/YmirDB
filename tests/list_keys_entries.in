list entries
list keys
set a 123 123 22 33
set b a 22 11 44
set c a b 22 11
list entries
list keys
bye
