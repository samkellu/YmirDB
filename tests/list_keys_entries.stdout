==774== Memcheck, a memory error detector
==774== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==774== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==774== Command: ./ymirdb
==774== 
==774== error calling PR_SET_PTRACER, vgdb might block
> no entries

> no keys

> ok

> ok

> ok

> c [a b 22 11]
b [a 22 11 44]
a [123 123 22 33]

> c
b
a

> bye
==774== 
==774== HEAP SUMMARY:
==774==     in use at exit: 0 bytes in 0 blocks
==774==   total heap usage: 50 allocs, 50 frees, 3,112 bytes allocated
==774== 
==774== All heap blocks were freed -- no leaks are possible
==774== 
==774== For lists of detected and suppressed errors, rerun with: -s
==774== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
